src/components/Notifications.tsx

```diff
@@ -1,56 +1,47 @@
-import { useMemo, useState } from "react";
-import useOnclickOutside from "react-cool-onclickoutside";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { beamerInitialize } from "react-beamer";
+import { useEffect } from "react";
+import { User } from "../model";
+import { defaultLanguage } from "./i18n/MenuIntlProvider";
+import { useAppConfiguration } from "../AppConfiguration";
 
-interface NotificationsProps {
-  notifications: ReadonlyArray<string>;
-  emptyNotificationText: string;
-}
+// this is a temporary beamer account with limited functions
+export const DEFAULT_BEAMER_ID = "wCWRrvwa50706";
```

Comment:
This value should not be here.

Alternatives could be:
* Adding a default configuration file [like this](https://github.com/FromDoppler/doppler-editors-webapp/blob/main/src/default-configuration.ts)
* Setting this in the configuration of your local environment [like here](https://github.com/FromDoppler/doppler-menu-mfe/blob/main/public/index.html#L30)

Reply:
in all place where the menu MFE is used, I give it of beamerId [like this](https://github.com/FromDoppler/doppler-editors-webapp/blob/main/src/default-configuration.ts). Do you think I can remove this line that you are telling me about?

Reply:
I think that this line can be safely removed because you will have the ID from the default configuration. Am I missing something?

Reply:
it's correct!
here, I give the ID in the Webapp: https://github.com/FromDoppler/doppler-webapp/pull/2199/commits/d9ef288af7dabd7f872e538fd3338279e75eae2c

in Doppler: https://github.com/MakingSense/Doppler/pull/9891/commits/275bb21382123371cf6bf3691f3ac4d2b5a2375d